### ISSUE TITLE
Tertiary button on default Drawer component fails Axe colour contrast 

### ISSUE THREAD

**Issue by PaulSumner-Sage** [Sage/carbon#4384]
### Current behaviour

Using the default Drawer component, with a default tertiary button to "filter" as seen in the Carbon Storyboard: https://carbon.sage.com/?path=/story/drawer--side-view-navigation Axe report fails contrast of the button.

### Expected behaviour

An alternative colour given for either Drawer background or Tertiary buttons recommended to avoid this contrasting issue.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/drawer--side-view-navigation

### Suggested Solution

Changing the default Drawer background shade to match with the default Sage colours for buttons may be a solution, or an advised alternative colour scheme for inside drawer?

### Carbon Version

84.1.0

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

MacOS, Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nicktitchmarsh** [comment]
@tempertemper @harpalsingh I think there needs to be a redesign of the tertiary button style in regards to accessibility but maybe just a review of how and where we use tertiary buttons?

**Comment by tempertemper** [comment]
Agreed. Tertiary buttons don't have any visual affordance other than colour that communicates they're clickable, so we'd fall short of the (Use of Color success criterion)[https://www.w3.org/TR/WCAG21/#use-of-color]. I suspect a redesigned tertiary button might be a while in the pipeline, so don't let that stop a colour contrast fix being put in place here 👍

**Comment by sjohnson-sage** [comment]
@harpalsingh it looks like we might need a slightly different colour for either the tertiary button text or the grey background, would you mind having a look please?

**Comment by clairedenning** [comment]
From the DS team: The drawer background can be white. That would fix the contrast issues. Is that an acceptable solution?

**Comment by harpalsingh** [comment]
@clairedenning we can not do that, as it would impact the design of Drawer. As mentioned elsewhere, we can handle ALL the Mint issues here by bumping the base color of Mint to pass the minimum accessbility on the app background color. I can handle that if we want that to the be the solution or do we simply just wait for Jade to be applied which handles that from the start, but we can not just change the background of Drawer so no that would not be an acceptable solution and actually would impact the reason why the drawer was given the color background it was.

**Comment by clairedenning** [comment]
Ah ok, in which case either bump up the Mint to increase the contrast, or wait for the new Jade :)

**Comment by harpalsingh** [comment]
Having look at the new color, it would not also fix this issue, so I suggest we bump the colors to pass, or we would need to carefully review drawer colors to consider if they can be lighter but not full white.

**Comment by DlgSHi** [comment]
Could @harpalsingh you provide the colours?

**Comment by harpalsingh** [comment]
@DlgSHi I spoke with Clair about this and what I will do is provide the value for "text" when used with Mint, and we may need to consider all the components that need this update.
